packages/http-server/src/__tests__/server.oas.spec.ts

```diff
@@ -206,7 +206,7 @@ describe.each([['petstore.no-auth.oas2.yaml', 'petstore.no-auth.oas3.yaml']])('s
     });
 
     // oas2 does not support overriding servers and named examples
-    if (file === 'petstore.oas3.json') {
+    if (file === 'petstore.no-auth.oas3.yaml') {
```

Comment:
I think you'd need to update lines 178-179 to _.no-auth_ too, don't you?

Reply:
I've put it in a constant so we stop with these magic strings.